airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/ConnectorDocumentationLayout.module.scss

```diff
@@ -34,6 +34,7 @@
 .rightPanelContainer {
   background-color: colors.$white;
   height: 100%;
+  overflow-y: scroll;
```

Comment:
Only nit: could we maybe add a comment into that file referencing the original issue, like: "Prevent the flickering described in <link>" so we more easily see in the code, why this is there (without needing to git blame to the PR).

Reply:
I pushed some new changes which solve both the original flickering problem in your issue, as well as the scrollbar flickering while resizing problem that we discussed over zoom. See the explanation in my comment here: https://github.com/airbytehq/airbyte/pull/15996#issuecomment-1229019827